hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/server/TestSecureContainerServer.java

```diff
@@ -125,6 +127,7 @@
   @BeforeClass
   public static void setup() throws Exception {
     DefaultMetricsSystem.setMiniClusterMode(true);
+    ExitUtils.disableSystemExit();
```

Comment:
The changes in this PR make sense to me, expecially the change inside MiniOzoneCluster, so that if you use a mini-ozone cluster it automatically calls `disableSystemExit()` and individual tests don't need to worry about it.

However the change like this one gives me a little bit of concern. How easy would it be for someone to add a new test class that calls some code with a `System.exit()` they have no idea about? Have we any other way to make this problem invisible to tests somehow? Perhaps there is no other feasible way to fix this, but thought I would ask anyway.

Reply:
> How easy would it be for someone to add a new test class that calls some code with a System.exit() they have no idea about? Have we any other way to make this problem invisible to tests somehow?

FindBugs / SpotBugs should catch most of such  (`DM_EXIT` rule).

Reply:
OK - makes sense. I guess `DM_EXIT` would catch any new instances of System.exit(), but I'm not sure it would catch a case where a new test invokes existing code which already has a System.exit() in place. However there probably isn't much we can do about that. So I am happy with the changes here.